sdk/react/discoveryPanel.js

```diff
@@ -198,6 +199,39 @@ var DiscoveryPanel = React.createClass({
         </Text>
       </View>);
   },
+
+  renderError: function() {
+    var errorTitleText = "SOMETHING NOT RIGHT! THERE SHOULD BE VIDEOS HERE.";
+    var errorContentText = "(Try Clicking The Discover Button Again On Reload Your Page)";
+    if (this.props.width < this.props.height) {
+      errorTitleText = "SOMETHING NOT RIGHT! THERE SHOULD\n BE VIDEOS HERE.";
```

Comment:
This isn't the best way to do word wrapping.  there must be better ways to do this.  how will this work in smaller views?